src/SFA.DAS.AssessorService.Application.Api/Validators/EpaOrganisationValidator.cs

```diff
@@ -28,7 +28,10 @@ public class EpaOrganisationValidator: IEpaOrganisationValidator
         private readonly IRegisterQueryRepository _registerQueryRepository;
         private readonly IStringLocalizer<EpaOrganisationValidator> _localizer;
         private readonly ISpecialCharacterCleanserService _cleanserService;
-       
+
+        private const string CompaniesHouseNumberRegex = "[A-Z0-9]{2}[0-9]{6}";
```

Comment:
Consider whether this needs to allow lower case characters in the first 2 values.  And then convert the save point to uppercase?  Makes it less nitty for users.
I'll leave that to your discretion, not going to not approve because of this.

Reply:
Sounds good.  Probably good to move to testing once done then.